docs/DECISIONS.md

```
@@ -316,12 +316,42 @@ Use **Tailwind's class-based dark mode** (`darkMode: 'class'`) with a `dark` cla
 
 ---
 
+## ADR-012: astro-pagefind Integration for Search
+
+**Date**: 2026-02-05
+**Status**: Accepted
+
+### Context
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)